Export.as

```diff
@@ -29,6 +29,20 @@ namespace VehicleState
 	//  3 = Rear Right
 	import float GetWheelDirt(CSceneVehicleVisState@ vis, int w) from "VehicleState";
 
+	// Get wheel falling state, and if in water. For w, use one of the following:
+	//  0 = Front Left
+	//  1 = Front Right
+	//  2 = Rear Left
+	//  3 = Rear Right
+	import FallingState GetWheelFalling(CSceneVehicleVisState@ vis, int w) from "VehicleState";
+
+	// Get the last turbo level that the vehicle touched.
+	import TurboLevel GetLastTurboLevel(CSceneVehicleVisState@ vis) from "VehicleState";
```

Comment:
Can you expand on what "last" means in this context? Is this value set even when you're not touching a turbo surface?

Reply:
Yes, if no turbo has been touched it will stay at 0, then once a turbo is touched the value is always 1-5, whichever one was the latest one. This applies for gates as well

Reply:
Might wanna update the comment to reflect that then:
```asc
// Get the last turbo level that the vehicle touched. This will return the last level even if the vehicle is not currently on a turbo surface.
```